pyhf/optimize/opt_tflow.py

```diff
@@ -35,18 +53,35 @@ def constrained_bestfit(self, objective, constrained_mu, data, pdf, init_pars, p
         nuis_cat = self.tb.concatenate(nuis_pars)
         pars = self.tb.concatenate([nuis_cat[:0],poi_par,nuis_cat[0:]])
         objective = objective(pars,data,pdf)
-        hessian   = tf.hessians(objective, nuis_cat)[0]
+        hessian   = tf.hessians(objective, nuis_cat)[0]+1e-10
```

Comment:
I'm not sure we really want to to this "dark" modification to the hessian that nobody knows is happening. I'd rather try to understand why the fit fails (if we can't figure it out, we should at least print a warning once that we are modifying the hessian). For now, I'd say we should remove this. It was just part of my testing